src/test/java/org/traccar/protocol/Gl200TextProtocolDecoderTest.java

```diff
@@ -48,6 +52,18 @@ public void testDecode() throws Exception {
 
         verifyAttributes(decoder, buffer(
                 "+BUFF:GTSTC,410301,864802030022424,,,0,,,,,,,0228,0002,4EE8,1BFF489,00,20181207134332,EC90$"));
+        
+        Position position = (Position) decoder.decode(null, null, buffer(
```

Comment:
For this test case I think it makes sense to add a new testing method. Something like `verifyNetwork`. I think we can start with an option that takes a single `CellTower`. That's the most common case.